content/docs/0.6.0/reference/troubleshooting/index.md

```diff
@@ -112,16 +112,49 @@ Please select the **Kubenet network plugin (basic)** when setting up your AKS cl
 </p></details>
 
 
-## Troubleshooting the Installer
+## Broken Keptn project
 
-In some cases the installer is not running correctly or crashes.
+When creating a project fails, this can cause a problematic state that manifests in a way that it cannot be deleted and the Keptn Bridge does not show any project.
```

Comment:
Deleting of a project should now be always possible.

Reply:
Thanks for the hint. I have changed the troubleshooting section to consider this.